ChangeLog.d/bugfix_1.txt

```diff
@@ -0,0 +1,3 @@
+Bugfix:
+    * x509: do not include time.h without MBEDTLS_HAVE_TIME.
```

Comment:
We usually acknowledge contributors in changelog entries. Unless you don't want to be credited, feel free to add

> Reported and fix contributed by naynajain in #3449.

You can use your name instead of your GitHub handle if you prefer.